policies.go

```diff
@@ -767,6 +775,10 @@ func (p *ybPartitionAwareHostPolicy) Pick(qry ExecutableQuery) NextHost {
 
 	start := tablesplitmetadeta.Floor(key)
 	replicas = (tablesplitmetadeta.getHosts(start))
+	for k := 0; k < len(replicas); k++ {
+		connectaddressreplicas = append(connectaddressreplicas, replicas[k].connectAddress)
+	}
+	log.Debug().Msgf("replicas for routing key %d are %v", key, connectaddressreplicas)
```

Comment:
Can move this inside a debug level check and keep it just once after CL==ONE check below

Reply:
Cannot check if debug logs are enabled in zerolog, I have updated the log to only print the first replica after shuffling of replicas when CL==ONE.